### ISSUE TITLE
latest multifunctional breaks tests

### ISSUE THREAD

**Issue by tngTUDOR** [brightway-lca/brightway2-io#276]
# Current
tests with `multifunctional-0.7.1` break tests with **'MatrixLabels' object has no attribute 'multifunctional_node_default'**

```
=================================================== short test summary info ===================================================
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_basic - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_ignore_multifunctional - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_skip_nonqualifying - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_duplicate_exchanges - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_inherit_process_location - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_inherit_process_unit - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_inherit_process_location_when_searching - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_get_default_global_location - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
FAILED tests/strategies/test_products.py::test_create_products_as_new_nodes_dataset_must_have_database_key - AttributeError: 'MatrixLabels' object has no attribute 'multifunctional_node_default'
```

# Expected
adapt codebase to match the api of multifunctional package

**Comment by cmutel** [comment]
Fixed in `bw2data` 4.0.dev53

**Comment by tngTUDOR** [comment]
~@cmutel , the `bw2data` package didn't get deployed `(╥﹏╥)`~
~https://github.com/brightway-lca/brightway2-data/actions/runs/10832232068/job/30056066529 ~

Nevermind, here it is:

https://pypi.org/project/bw2data/4.0.dev53/
